src/components/LayoutHeader/Nav/LinkItems/index.tsx

```diff
@@ -102,18 +106,25 @@ const LinkItems: React.FC = () => {
       {navLinkItemsData.map((item, i) => {
         const popup = isPopup(item) ? popups[item.popupName] : undefined
         return (
-          <li key={i} className={styles.linkItem} ref={popup?.containerEl}>
+          <li
+            key={i}
+            className={styles.linkItem}
+            ref={popup?.containerEl}
+            onMouseEnter={isMobile ? undefined : popup?.open}
```

Comment:
Instead of adding a package to detect mobile users, I wonder if we should update our popup logic to open on `:focus-within` instead of toggling `button.onClick` 🤔. Then mobile users wouldn't have to double tap... but this code works just fine and we can always update it later :)